tests/phpunit/Integration/JSONScript/TestCases/p-0414.json

```diff
@@ -324,8 +324,8 @@
 			"about": "#13 `.888499949` is being rounded to `.888500`",
 			"subject": "Example/P0414/7a",
 			"skip-on": {
-				"hhvm-*": "Avoid .888500 vs. .888499 msec issue due to hhvm#6899"
-			},
+ 				"php": [">7.1", "Avoid .888500 vs. .888499 msec issue due to https://bugs.php.net/bug.php?id=76822"]
```

Comment:
No need to check for > 7.1 anymore

Reply:
How so? Without this the test fails due to https://bugs.php.net/bug.php?id=76822

Reply:
My bad, should have been more clear. Since we don't support any PHP versions under 7.1, do we still need the test case?

Reply:
Oh I see!

Reply:
Thanks, done.